app/lib/actions/katello/agent_action.rb

```diff
@@ -49,6 +49,7 @@ def run(event = nil)
 
             schedule_timeout(timeout)
           end
+        when Dynflow::Action::Skip # rubocop:disable Lint/EmptyWhen
```

Comment:
if you add a comment inside the when block here you can a) avoid disabling this particular lint check b) make some future bug-fixer who has to read this code grateful for your comment explaining what this thing is (your PR name pretty much articulates that!)

Reply:
Updated with a comment :)